### DISCUSSION TITLE
Cant get this to work

### DISCUSSION THREAD

**Discussion by Zonata** [gregsullivan/_tw#95] [Q&A]
Hi I have followed the instructions but I can not get the css to be applied to the element.  Not sure what I have missed.

**Comment by gregsullivan** [comment]
Hi—Thanks for posting this!

When you say you can't get the CSS to be applied to the element, what CSS or Tailwind classes are you trying to add, and in what file?

What is showing in your terminal window with `npm run watch` running?
